mobile/src/main/java/ch/epfl/sdp/mobile/ui/social/SocialScreen.kt

```diff
@@ -244,14 +247,14 @@ fun <P : Person> SearchResultList(
     ) { player ->
       PersonItem(
           person = player,
-          modifier = modifier.clickable { onShowProfileClick(player) }.animateItemPlacement(),
+          modifier = modifier,
```

Comment:
Should be 
```suggestion
          modifier = Modifier.animateItemPlacement(),
```

with `Modifier` instead of `modifier` and you should keep `animateItemPlacement()`.